Makefile

```diff
@@ -30,6 +30,10 @@ DOCKER_NAME ?= kiali/kiali
 DOCKER_VERSION ?= dev
 DOCKER_TAG = ${DOCKER_NAME}:${DOCKER_VERSION}
 
+
+# Change that IfNotPresent to use on dev mode 
```

Comment:
I'm not entirely sure what this comment is saying... do you mean "Change the Always below to IfNotPresent when in dev mode" or something like that?

In other words, "Change that IfNotPresent" is confusing because there is no IfNotPresent here, so what is "that IfNotPresent" referring to?

Reply:
BTW: if this does mean "PULL POLICY should be set to IfNotPresent if in dev mode" then we can add a if-else here - look at the value of DOCKER_VERSION and if "dev" then we should set the PULL_POLICY (unless PULL_POLICY is already set - in which case use it since the user is telling us specifically what value to use).